index.html

```diff
@@ -0,0 +1,181 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css" href="styles.css" />
+    <link href="https://cdn.jsdelivr.net/npm/remixicon@2.5.0/fonts/remixicon.css" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter&display=swap" rel="stylesheet">
+    <script src="https://kit.fontawesome.com/b33cf67da8.js" crossorigin="anonymous"></script>    
+    <title>My First Portfolio</title>
+</head>
+<body>
+        <header class="welcome-message">
+            <a class="welcome">Welcome</a>
+            <a class="menu-icon"><i class="ri-menu-line"></i></a>
+        </header>
+        <section class="greetingsandsocial">
+            <h2 class="intro1">Hey there. I'm Edgard</h2>
+            <h2 class="intro2">I'm a software developer</h2>
+            <p class="intromessage"> I can help you build a product , feature or website. 
+            Look through some of my work and experience! 
+            If you like what you see and have a project you need coded, 
+            don't hestiate to contact me.</p>
+                <ul class="social-container">
+                    <li><a href="https://twitter.com/soza_edgard"><i class="fa-brands fa-twitter"></i></a></li>
+                    <li><a href="https://www.linkedin.com/in/edgard-soza-sobalvarro/"><i class="fa-brands fa-linkedin-in"></i></a></li>
+                    <li><a href="#"><i class="fa-brands fa-m"></i></a></li>
+                    <li><a href="https://github.com/edgardsoza"><i class="fa-brands fa-github"></i></a></li>
+                    <li><a href="#"><i class="fa-brands fa-angellist"></i></a></li>
+                </ul>
+        </section>
+    <section class="recent">
+        <img src="images/disabled.png" alt="">
+        <h1 class="recent-works">My Recent Works</h1>
+        <a><i class="fa-solid fa-horizontal-rule"></i></a>
+        <div class="project-container">
+            <div class="project1">
+                    <img class="projectimages" src="images/project1.jpg" alt="">
+                    <div class="project-content">
```

Comment:
Kindly fix the margins of your application to match the figma design. This was a requirement for this milestone
<img width="1438" alt="Screenshot 2023-01-15 at 03 52 33" src="https://user-images.githubusercontent.com/48546992/212503789-2ca6118f-1dd6-45d2-ab0c-5ca8d414a16b.png">
This is your design currently:
<img width="662" alt="Screenshot 2023-01-15 at 04 03 16" src="https://user-images.githubusercontent.com/48546992/212503864-7d780e3a-c99d-4b21-80a2-9eace4cb3329.png">
This is how the figma template looks like:
<img width="375" alt="Screenshot 2023-01-15 at 03 51 55" src="https://user-images.githubusercontent.com/48546992/212503884-a7b90a80-7eba-47fd-95ff-5776af0166f9.png">

Reply:
done